packages/app/source/server/projections/todos-projection.js

```diff
@@ -1,7 +1,7 @@
 Space.eventSourcing.Projection.extend('Todos.TodosProjection', {
 
-  collections: {
-    todos: 'Todos.TodoLists'
+  dependencies: {
```

Comment:
Ok, no we'll need to make some changes as it's more hard wired than I thought. It's time to review how `ProjectionRebuilder` can be reworked